server/template/paste.xhtml

```diff
@@ -0,0 +1,61 @@
+<html py:extends="master.xhtml" py:strip="True">
+<body>
+
+  <head py:block="head" py:strip="True">
+    <title py:block="master_title">Paste Logs</title>
+  <style type="text/css">
+   .container {
```

Comment:
I am pretty sure these styles can be achieved purely with Bootstrap. Use the `row` class to achieve the `clear: both;` functionality. Also, use `col-md-#` where `#` is a number to configure width.

If you cannot achieve this with Bootstrap, please add these styles to the `resources/sass/_widb.sass`. You will need to add a more unique class to the `div` below:

    <div class="container upload-container">

And then in `_widb.sass` you can use:

    .upload-container {
        width: 1000px;
        clear: both;

        input: {
            width: 100%;
            clear: both;
        }
    }